subsys/debug/asan.c

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2019 Jan Van Winkel <jan.van_winkel@dxplore.eu>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <sanitizer/asan_interface.h>
```

Comment:
Is this include still needed?

Reply:
It is not a hard requirement, but it contains the declaration  __asan_default_options

Reply:
Ok.. my thinking was that if this header is renamed or moved in any version of the library it would break for whoever faces it. While the declaration below would not cause trouble unless the function types themselves would change.
So it would seem safer to not have this include line?

Reply:
removed include file